exercises/sublist/sublist.spec.js

```diff
@@ -0,0 +1,126 @@
+var List = require('./sublist');
+
+
+describe('sublist', function() {
+
+  it('two empty lists are equal', function() {
+    const listOne = new List();
```

Comment:
What about using `var` instead?

In this case, I don't have a strong opinion. We should try to use just ES5 code in this track. But, `const` is not a very advanced, or a very complex ES6 feature. In fact, it's a good first step towards ES6.

If it was a destructuring, I's strongly recomend going back to ES5. But it's a `const`, not a big deal.

What do you think? Should we stick to ES5 even in this case?